public/omni/getting-started/how-to-install-talosctl.mdx

```diff
@@ -54,3 +54,11 @@ However, users with at least the [Operator](https://omni.siderolabs.com/referenc
 | `talosctl restart`                             |
 | `talosctl service <id> [stop\|start\|restart]` |
 | `talosctl shutdown`                            |
+
+## Sensitive resources
+
+Omni prevents viewing machine configuration which might contain sensitive PKI. To view the list of resources that cannot be viewed you can use this command:
```

Comment:
i added this comment yesterday but for some reason it has been pending since, but here are my thoughts:

i personally think that there are too many "view" in this paragraph

would this be a better rewrite?

'Omni restricts access to machine configurations that may contain sensitive PKI information. To see which resources are restricted, run the following command:'